### PR TITLE
Added metadata updater script

### PR DESCRIPTION
**Pull request by dpopleton**

### CODE REVIEW

**Comment by dpopleton** [comment]
Please be thorough Marc, as I am not confident in packages yet.

File: src/scripts/metadata_update.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import argparse
+import re
+from sqlalchemy.engine.url import make_url
+from ensembl.production.metadata.updater import CoreMetaUpdater
+def main():
+    parser = argparse.ArgumentParser(description='Method for loading data from a spcific database to a metadata registry')
```
Comment (marcoooo) [inline src/scripts/metadata_update.py:19]:
```suggestion
    parser = argparse.ArgumentParser(description='Method for loading data from a specific database to a metadata registry')
```

File: src/scripts/metadata_update.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    parser.add_argument('-m', '--uri', help='Metadata database URI', required=True)
+    parser.add_argument('-d', '--database_uri', help='URI of database to load', required=True)
+    parser.add_argument('-r', '--release', help='Release')
```
Comment (marcoooo) [inline src/scripts/metadata_update.py:23]: Not sure we need to set this, is it meant to update an already released genome data? I think we said that the release processing would be independent from handover, so we'll only create the dataset and assign release later in the processing.

File: src/scripts/metadata_update.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    elif re.match('^\w+_?\d*_\d+$', db_url.database):
+        raise Exception("other not implemented yet")
+    elif re.match(
+            '^ensembl_accounts|ensembl_archive|ensembl_autocomplete|ensembl_metadata|ensembl_production|ensembl_stable_ids|ncbi_taxonomy|ontology|website', db_url.database):
+        raise Exception("other not implemented yet")
+    else:
+        raise "Can't find data_type for database " + db_url.database
+
+# !DP!#
+# NOT DONE#######################Worry about this after the core et al are done. Don't delete it yet.
```
Comment (marcoooo) [inline src/scripts/metadata_update.py:54]: Could you already implement it in the script, remove this "internal" comment from a public repo. We'll know well that collection db are not handled yet. And the error should be returned from api

File: src/scripts/metadata_update.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    main()
+
+
+
+
+
+
+
+
+
```
Comment (marcoooo) [inline src/scripts/metadata_update.py:70]:
```suggestion
```

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: src/scripts/metadata_update.py (hunk 2 above)
Comment (marcoooo) [reply src/scripts/metadata_update.py:23]: I was actually talking about the `--release` param, the others are fine. Assuming those are "r/w" connexion strings, right?

Comment (marcoooo) [reply src/scripts/metadata_update.py:23]: We'll keep it for the future then.

**Pull request closed without merging**